projects/laji/src/app/+project-form/project-form.component.ts

```diff
@@ -46,14 +46,19 @@ export class ProjectFormComponent implements OnInit, OnDestroy {
     private projectFormService: ProjectFormService,
     private formPermissionService: FormPermissionService,
     public userService: UserService,
-    private router: Router
+    private router: Router,
+    private browserService: BrowserService
 ) {}
 
   vm$: Observable<ViewModel>;
   formPermissions$: Observable<FormPermission>;
   showNav$: Observable<boolean>;
   isPrintPage$: Observable<boolean>;
   redirectionSubscription: Subscription;
+  isDesktopScreen: boolean;
+  isNavabarToggledSubject: Subject<boolean> = new BehaviorSubject<boolean>(false);
+  isNavabarToggled$ = this.isNavabarToggledSubject.asObservable();
+  subscriptionShowNav: Subscription;
```

Comment:
1. Fix typo (Nava -> Nav)
2. Are you sure we can't just use `subscriptionShowNav` directly as an input observable for `showNav$`? if we can't then maybe it'd be cleaner to do the `asObservable()` conversion at directly at the `showNav$` initialization, so we wouldn't have that many instance variables.